akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -73,11 +73,11 @@ class Http2ServerSpec extends AkkaSpecWithMaterializer("""
           response:                    HttpResponse,
           expectedResponseHeaderBlock: ByteString
         ): Unit = {
-          sendHEADERS(streamId, endStream = true, endHeaders = true, requestHeaderBlock)
-          expectRequest() shouldBe expectedRequest
+          network.sendHEADERS(streamId, endStream = true, endHeaders = true, requestHeaderBlock)
+          user.expectRequest() shouldBe expectedRequest
 
-          responseOut.sendNext(response.addAttribute(Http2.streamId, streamId))
-          val headerPayload = expectHeaderBlock(streamId)
+          user.responseOut.sendNext(response.addAttribute(Http2.streamId, streamId))
```

Comment:
Is that the same as `user.emitResponse`? Maybe that can be changed then as well.